New version adapted to nullable columns and null handling in multi-stage engine (#304)
pinot-contrib/pinot-docs: reference/multi-stage-engine.md

Apache Pinot 1.0 Multi-Stage Query Engine overview
{% endembed %}

## How does the multi-stage handle nulls?
Before Pinot 1.1.0, the multi-stage query engine did not support null handling, treating all columns as non-nullable.

> In Pinot 1.1.0, the multi-stage query engine supports null handling if and only column base null storing is enabled.

Reviewer: ```suggestion
In Pinot 1.1.0, the multi-stage query engine supports null handling if column based null storing is enabled.
```

> You can read more about that in the [Null value support](../developers/advanced/null-value-support.md) section.

Reviewer: ```suggestion
For more information, see the [Null value support](../developers/advanced/null-value-support.md) section.
```

## Why use the multi-stage query engine?

> You must use the multi-stage query engine (v2) to query distributed joins, window functions, and other multi-stage operators in real-time.&#x20;

Reviewer: ```suggestion
You must use the multi-stage query engine (v2) to query distributed joins, window functions, and other multi-stage operators in real-time.
```
